lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -71,7 +70,8 @@ def validate_twilio_event(event: dict) -> bool:
 
         validator = RequestValidator(auth_token)
         uri = f"https://{event['headers']['host']}/vanotify/twoway/vettext"
-        decoded = base64.b64decode(event.get('body')).decode()
+
+        decoded = base64.b64decode(event.get('body')).decode('utf-8')
```

Comment:
I have added utf-8 for consistency sake. The message sending method decodes this way.